src/Countries/Syria.php

```diff
@@ -0,0 +1,298 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use RuntimeException;
+
+class Syria extends Country
+{
+    /*
+     * > No library or built-in php intl functions convert dates properly for all years or all country including “geniusts/hijri-dates”.
+     *
+     * > It is most logical to prepare the dates between 1970 and 2037 as a constant property for Islamic holidays, Because Islamic holidays predicted and actual dates may change until the last moment.
+     *
+     * > Since the information on wikipedia is incorrect, it was obtained by searching the old calendar on Google Images for each year.
+     *
+     * > The accuracy of the information has been double-checked.
+     *
+     * > A converter algorithm that will cover all years does not seem possible.
+     */
+
+    public const alFitrHolidays = [
+        1970 => '12-01',
```

Comment:
Have you verified all dates or just copy pasted from Turkey?

Reply:
All Islamic countries share the same holiday dates based on the Hijri calendar, which is determined by the movements of the moon. That's why holidays in Syria and Turkey coincide.